scripts/macos/apple_compile.sh

```diff
@@ -99,6 +99,37 @@ if [[ "$OS" == "ios" ]]; then
   print G "done."
 fi
 
+if [[ "$OS" == "ios" ]]; then
+  printn Y "Glean stuff... \n"
+  
+  # Version MUST be kept in sync with Glean's version. We should probably automate the syncing.
+  cargo install uniffi_bindgen@0.19.6
+  mkdir -p ios/Glean
+  # Generate the FFI stuff for the Glean iOS SDK
+  uniffi-bindgen generate -l swift -o "ios/Glean" "3rdparty/glean/glean-core/src/glean.udl"
```

Comment:
If you vendor glean anyway, you should be able to use the `embedded-uniffi-bindgen` crate and thus not need to sync the uniffi-bindgen version:

```suggestion
cargo run --manifest-path 3rdparty/glean/tools/embedded-uniffi-bindgen/Cargo.toml -- generate -l swift -o "ios/Glean" "3rdparty/glean/glean-core/src/glean.udl"
```

Reply:
Nice!